mltsp/featurize.py

```diff
@@ -20,26 +22,40 @@ def write_features_to_disk(featureset, featureset_id):
 def load_and_store_feature_data(features_path, featureset_id="unknown",
                                 first_N=None):
     """Read features from CSV file and store as xarray.Dataset."""
-    targets, metadata = ft.parse_headerfile(features_path)
+    targets, meta_features = ft.parse_headerfile(features_path)
     if first_N:
-        metadata = metadata[:first_N]
+        meta_features = meta_features[:first_N]
         if targets is not None:
             targets = targets[:first_N]
-    featureset = ft.assemble_featureset([], targets, metadata)
+    featureset = ft.assemble_featureset([], targets, meta_features)
     write_features_to_disk(featureset, featureset_id)
     return featureset
 
 
-def featurize_task_params_list(ts_paths, features_to_use, metadata=None,
+def featurize_data_task_params(times, values, errors, labels,
+                               features_to_use, meta_features=None,
+                               custom_script_path=None, custom_functions=None):
+    """Create list of tuples containing params for `featurize_data_task`."""
```

Comment:
Could I just say "refer to `featurize_time_series`"? The arguments are the same but the descriptions are long/complicated. Same with the function below.